src/applications/claims-status/sass/claims-status.scss

```diff
@@ -1257,4 +1257,32 @@ $marker-text-width: 5.625rem;
   }
 }
 
+va-icon.phase-complete {
```

Comment:
Is this css being used?

Reply:
yep this gets used on line 124. Its what makes the checkbox icons green.